src/components/read-only-flashcard/read-only-flashcard.scss

```diff
@@ -9,6 +9,12 @@ $card-height: 175px;
   width: 100%;
   gap: 20px;
   margin: 10px 0px;
+
+  @media screen and (max-width: $medium-screen) {
```

Comment:
nice! but I still think it manages to get too small, perhaps we should use `$large-screen` (or even something in between lg/giant, idk)

Reply:
yeah, I agree it gets too small. 

What do you think about breaking the navbar earlier? 

I've noticed when I have my phone horizontal, the sidebar doesn't break and takes up about a third of the screen. 

### Right before the nav breaks when nav breaks on large and cards break on medium 

![Screen Shot 2022-09-23 at 11 43 44 AM](https://user-images.githubusercontent.com/39753553/192026473-31d68dcd-dec1-44f4-b299-a9999856f449.png)

### Right before the cards break when nav breaks on large and cards break on medium 

![Screen Shot 2022-09-23 at 11 43 29 AM](https://user-images.githubusercontent.com/39753553/192026459-0e25dae6-4089-4b6a-95aa-37fa0a48bd06.png)

Reply:
so like this? not a bad idea at all.

```diff
 .sidebar-layout-sidebar {
-  @media screen and (max-width: $large-screen) {
+  @media screen and (max-width: $giant-screen) {
     padding-left: unset;
     width: $sidebar-width / 1.15;
   }

-  @media screen and (max-width: $medium-screen) {
+  @media screen and (max-width: $large-screen) {
     display: none;
   }
 }
@@ -47,11 +47,11 @@ $content-height: calc(100vh - #{$header-height + $content-padding});
   margin-left: $sidebar-width;
   padding-top: $header-height;

-  @media screen and (max-width: $large-screen) {
+  @media screen and (max-width: $giant-screen) {
     margin-left: ($sidebar-width / 1.15);
   }

-  @media screen and (max-width: $medium-screen) {
+  @media screen and (max-width: $large-screen) {
     margin-left: 0;
   }
```

we would need to change breakpoints for the navigation hamburger `.c-header-content` and some other things if we're hiding the sidebar early though

Reply:
Ok, I think I got it all updated. How does this look?

https://user-images.githubusercontent.com/39753553/192109663-7ea3131f-57a4-4bc8-8bfe-31dcb69f3486.mov

Reply:
this looks great!